a11y-meta-display-guide/2.0/draft/localizations/en-US/display_guide_vocabulary_w3c_en-US.json

```diff
@@ -6,269 +6,316 @@
-            "conformance-level-aa": " Level AA",
-            "conformance-level-a": " Level A",
-            "conformance-certification-info-compact": "The publication was certified",
-            "conformance-certification-info-descriptive": "The publication was certified",
-            "conformance-certification-date-prefix": " on ",
-            "conformance-certifier-prefix-compact": " by ",
-            "conformance-certifier-prefix-descriptive": " by ",
-            "conformance-certifier-credentials-prefix-compact": " The certifier's credential is ",
-            "conformance-certifier-credentials-prefix-descriptive": " with a credential of ",
-            "conformance-certifier-report": "For more information refer to the certifier's report."
-        }
-    },
-    "pre-recorded-audio": {
-        "pre-recorded-audio-title": "Pre-recorded audio",
-        "pre-recorded-audio-only": {
-            "descriptive": "Audiobook with no text alternative",
-            "compact": "Audio only"
-        },
-        "pre-recorded-audio-complementary": {
-            "descriptive": "Contents available as complementary audio and text",
-            "compact": "Complementary audio and text"
-        },
-        "pre-recorded-audio-synchronized": {
-            "descriptive": "All the content is available as pre-recorded audio synchronized with text",
-            "compact": "Synchronized audio and text"
-        },
-        "pre-recorded-audio-no-metadata": {
-            "descriptive": "No information about pre-recorded audio",
-            "compact": "No information about pre-recorded audio"
+        "conformance-aaa": {
+            "compact": "This publication exceeds accepted accessibility standards",
+            "descriptive": "The publication contains a conformance statement that it meets the EPUB Accessibility and WCAG 2 Level AAA standard"
+        },
+        "conformance-certification-info": {
+            "compact": "The publication was certified on",
+            "descriptive": "The publication was certified on"
+        },
+        "conformance-certifier": {
+            "compact": "The publication was certified by",
+            "descriptive": "The publication was certified by"
+        },
+        "conformance-certifier-credentials": {
+            "compact": "The certifier's credential is",
+            "descriptive": "The certifier's credential is"
+        },
+        "conformance-certifier-report": {
+            "compact": "For more information refer to the certifier's report",
+            "descriptive": null
+        },
+        "conformance-claim": {
+            "compact": "This publication claims to meet",
+            "descriptive": null
+        },
+        "conformance-details": {
+            "compact": "Detailed conformance information",
+            "descriptive": null
+        },
+        "conformance-epub-accessibility-1-0": {
+            "compact": "EPUB Accessibility 1.0",
```

Comment:
The whitespace needed to join these strings together is missing in the new output.

Reply:
Right, I currently use the `normalize-space` function because there are multiple spaces inside the extracted strings. I need to reason how to remove the multiple spaces, keeping the leading and trailing spaces.

Reply:
I think I can find a solution, even when the software reformats the file.